### PR TITLE
Fixed a typo

### PR DESCRIPTION
**Pull request by bkcarroll**

### CODE REVIEW

**Comment by PRMerger19** [comment]
@bkcarroll : Thanks for your contribution to the Azure documentation! The author, @markingmyname, has been notified to review your proposed change.

File: docs/reporting-services/report-data/report-data-pane.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ manager: "kfile"
...
+  Use the **Report Data** pane to view the currently defined parameters, data sources, datasets, field collections, and images in your report. The Report Data pane displays a hierarchical view of the items that represent data in your report. The top level nodes represent built-in fields, parameters, images, and data source references. Expand each node to view the data items. For example, when you expand a data source node, the datasets defined for that data source appear. When you expand a dataset, its field collection appears. Drag items to the report design surface to link data with report items on the report page.
```
Comment (markingmyname) [inline docs/reporting-services/report-data/report-data-pane.md]: just a small suggestion.  I would change "top level" to "top-level" maybe?

Comment (MightyPen) [reply docs/reporting-services/report-data/report-data-pane.md]: I see why the addition of a dash, to replace the space in "top level", might be suggested.  But we should lean toward avoid unnecessary uses of dashes, and I feel that the text is clear and at least equally correct in this case without adding the dash.
Thanks for caring enough to chime in!  Seriously thanks.  :-) 
GeneMi at MS (MightyPen)

**Comment by MightyPen** [comment]
@craigg-msft  #sign-off

**Comment by PRMerger12** [comment]
@MightyPen: I'm sorry - only the author of this article, @markingmyname, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

**Comment by MightyPen** [comment]
BTWay 'bkcarroll',  It can help us review a bit more efficiently if the mis-typed word is named in the PR title, or in a comment.  Too often the GitHub differ does not highlight the one word that is changed within a large paragraph.  Thanks.

**Comment by MightyPen** [comment]
BTWay, the offending mis-typed word was 'dana', instead of the now corrected 'data'.

**Comment by v-shils** [comment]
Hi @MightyPen @markingmyname, please add the appropriate comment, so automation can take action.

File: docs/reporting-services/report-data/report-data-pane.md (hunk 1 above)
Comment (markingmyname) [reply docs/reporting-services/report-data/report-data-pane.md]: @MightyPen I approve. 👍

**Comment by Lisaco88** [comment]
Author approved in above convo. Merging now.

**Pull request merged**